pkg/cmd/admissioncontroller/main.go

```diff
@@ -142,14 +143,17 @@ func (wh *WebHook) Startup(certs *tls.Certificate) {
 	wh.server = &http.Server{
 		Addr: fmt.Sprintf(":%v", wh.port),
 		TLSConfig: &tls.Config{
-			MinVersion:   tls.VersionTLS12,
-			Certificates: []tls.Certificate{*certs}},
+			MinVersion:   tls.VersionTLS12,           // No SSL, TLS 1.0 or TLS 1.1 support
+			NextProtos:   []string{"h2", "http/1.1"}, // prefer HTTP/2 over HTTP/1.1
```

Comment:
Should we remove the cipher suits which may be treated as error by http/2 with tls 1.2?

see RFC 7540

https://datatracker.ietf.org/doc/html/rfc7540#appendix-A